swap_meet/vendor.py

```diff
@@ -1,2 +1,96 @@
+from queue import PriorityQueue
+from swap_meet.item import Item
+
 class Vendor:
-    pass
\ No newline at end of file
+    def __init__(self, inventory  = None):
```

Comment:
Based on PEP8 styling guide, "Don’t use spaces around the = sign when used to indicate a keyword argument, or when used to indicate a default value for an unannotated function parameter."

Reply:
Thank you for sharing this! I'm going to include it in my refactored code.